less/new-flow-test/shared.less

```diff
@@ -102,8 +102,6 @@ input[name="payment-type"] + label {
 input[name="payment-type"] + label {
   background-image: linear-gradient(transparent, rgba(0, 0, 0, 0.2));
   position: relative;
-  padding-top: 5px;
-  padding-bottom: 5px;
```

Comment:
The padding on number buttons looks centered, but with lower case text, it looked off.

So I'm splitting the padding bit out of the rule, and into separate specific ones.